tokenserver/tests/test_service.py

```diff
@@ -119,6 +121,21 @@ def test_discovery(self):
             }
         })
 
+    def test_version_returns_404_by_default(self):
+        self.app.get('/__version__', status=404)
+
+    def test_version_returns_file_in_current_folder_if_present(self):
+        content = {'version': '0.8.1'}
+        fake_file = mock.mock_open(read_data=json.dumps(content))
+        with mock.patch('os.path.exists'):
+            with mock.patch('tokenserver.views.open', fake_file, create=True):
+                response = self.app.get('/__version__')
+                self.assertEquals(response.json, content)
+
```

Comment:
nit: I don't know if you want to cover all this but you should also have:
* 404 if file missing
* version.json override from the 3 supported locations
* memoized result